### ISSUE TITLE
Lost session with FileBackend and slower latency storage (e.g. network storage like Ceph)

### ISSUE THREAD

**Issue by hlubek** [neos/flow-development-collection#1911]
- Concurrent access to the session can cause a re-login (the session looks expired to the user) when using a slower latency filesystem (e.g. network storage like Ceph RBD)
- We experienced this behaviour in a Neos installation with a persistent filesystem backed by a Ceph RBD persistent volume in a Kubernetes installation
- It looks like the file locking and wait operation (see https://github.com/neos/flow-development-collection/blob/fdb7b00603b3982025047bd84ad1b115bfe566a9/Neos.Cache/Classes/Backend/SimpleFileBackend.php#L495) is reporting the session cache entry as `false` after 3 retries with varying wait time - Neos presents a login screen if a single AJAX request was not authenticated
- Using a ephemeral, local storage, does solve the issue as a workaround
- It would be better if the wait time and retry count could be configurable as options, so it can be extended (retry count for a session should be unlimited or the entry should not be reported as missing but an exception could be thrown)

**Comment by kitsunet** [comment]
Yep, have seen that as well before, I think also the whole session cache entry system should get a close look for efficiency.
